mongo/interfaces/interfaces.go

```diff
@@ -40,6 +40,7 @@ type MongoDB interface {
 type Collection interface {
 	Find(query interface{}) Query
 	FindId(id interface{}) Query
+	Pipe(pipeline interface{}) Pipe
```

Comment:
Why was this function added? Did we update the mongo lib version?

Reply:
No, sorry it is fruit of a bad rebase. It is fixed!